http/telegraf.go

```diff
@@ -509,6 +553,10 @@ func (c *C) delete(urlPath string) *cReq {
 	return c.newClientReq(http.MethodDelete, urlPath, nil)
 }
 
+func (c *C) get(urlPath string) *cReq {
```

Comment:
Why create and use a new client type over using `traceClient`?

Reply:
its using traceClient under the hood to make the responses still 👍